src/model/helpers.rs

```diff
@@ -468,7 +597,7 @@ pub async fn get_subscription_watchers_for_account_by_app_or_all_app(
         SELECT project, did_key, sym_key
         FROM subscription_watcher
         LEFT JOIN project ON project.id=subscription_watcher.project
-        WHERE account=$1 AND (project IS NULL OR project.app_domain=$2)
+        WHERE expiry > now() AND account=$1 AND (project IS NULL OR project.app_domain=$2)
```

Comment:
This fixed query potentially returning to expired watchers, not a major issue, but a fix nevertheless